infra/Makefile

```diff
@@ -0,0 +1,7 @@
+ENVIRONMENT?=staging
+
+init:
+	cd $(ENVIRONMENT) && terraform init
```

Comment:
I initially thought this `&&` was unnecessary since `make` fails on the first non-zero exit, but unfortunately it seems like `make` executes each statement in its own subshell, so `cd` doesn't propagate the changed directory to subsequent statements. Yuck.

This `&&` is probably the cleanest way to do it then. I think a more typical pattern is to not use `cd` and to just invoke a Makefile at the root of the directory (either by cd-ing or by `make -C`), but this is a PITA for the staging + production setup (especially if we want to keep it DRY. Symlinking `Makefile` does work, but I think it's not worth the hassle).

Reply:
Yeap I just use cd xx && everywhere at work